<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Select Specialisation</string>
   </property>
  </widget>
  <widget class="QComboBox" name="specialisation_cb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1091</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Select Consultation Fee</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fee_cb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>1091</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Select Date &amp; Time</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="date_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>1091</width>
     <height>21</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Available Doctors</string>
   </property>
  </widget>
  <widget class="QListWidget" name="doctors_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>1091</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>730</y>
     <width>1091</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>810</y>
     <width>1091</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="book_appointment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>770</y>
     <width>1091</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Appointment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
